contents/articles/2016-10_michael-barz-ubicomp/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Michael Barz's demo at UBICOMP
+date: Mon Oct 17 2016 15:59:02 GMT+0700 (ICT)
+author: Pupil Dev Team
+subtitle: "Michael Barz's demonstration used Pupil Capture with custom plugin for gaze-guided object classification using deep neural networks for attention-based computing..."
+featured_img_thumb: "../../../../media/images/blog/theinformation_2016.png"
+---
+
+Michael Barz's demonstration used Pupil Capture with custom plugin for gaze-guided object classification using deep neural networks for attention-based computing. 
```

Comment:
Michael Barz and Daniel Sonntag use Pupil to create a gaze-guided object classification system. Check out the video below for a demonstration.